be-java-springboot/files/README.md

```diff
@@ -0,0 +1,59 @@
+# {{project-name}} - provisioned from the OpenDevStack Java/Spring Boot QuickStarter (be-java-springboot)
```

Comment:
If we want to have the name dynamically created, we need to process this file in the provisioning and use @component_id@ and @project_id@ instead

Reply:
Okay I see, I copied it from here: https://github.com/opendevstack/ods-quickstarters/blob/master/be-rust-axum/rust-template/README.md#project-name---from-the-opendevstack-rust-quickstarter-be-rust-axum
Not sure why it works there 😆 .
How do I process this file in the provisioning?

Reply:
You can check this as an example:

- https://github.com/opendevstack/ods-quickstarters/blob/master/e2e-etl-python/Jenkinsfile#L37-L41
- https://github.com/opendevstack/ods-quickstarters/blob/master/e2e-etl-python/prod.yml.template

Reply:
The doc (https://www.opendevstack.org/ods-documentation/opendevstack/latest/jenkins-shared-library/quickstarter-pipeline.html#_odsquickstarterstagerenderjenkinsfile) mentions specifically that this stage is for rendering the Jenkinsfile, I'm trying to render it in a README.md file though. I guess that would be quite a misuse of that stage in case it works at all, right? Maybe it is simpler just to replace it with something like `<project_id>` and don't render it?

Reply:
We already make use of that stage to render other files that are not the Jenkinsfile though :sweat_smile:

Reply:
Ok even though I'd suggest to rename this Rendering Stage then to make that clear, I used it now for the readme :)